### PR TITLE
inject js integration library

### PR DESCRIPTION
**Pull request by vitalikdanchenko**

### CODE REVIEW

**Comment by bogdan** [comment]
@vitalikdanchenko please assign @Tonkonozhenko for all tasks on talkable-ruby now

File: lib/talkable/middleware.rb [hunk 1]
```diff
@@ -66,19 +83,42 @@ def sync_uuid_content(uuid)
...
+    def integration_content
+      integration_script_content + integration_init_content
+    end
+
+    def integration_script_content
+      %Q{
+<script>
+  window._talkableq = window._talkableq || [];
+  _talkableq.push(['init', {
+    site_id: '#{CGI::escape(Talkable.configuration.site_slug)}',
```
Comment (Tonkonozhenko) [inline lib/talkable/middleware.rb]: Why do we need `CGI::escape` here?

Comment (Tonkonozhenko) [inline lib/talkable/middleware.rb]: Use CGI.escape instead CGI::escape please

File: lib/talkable/middleware.rb [hunk 2]
```diff
@@ -66,19 +83,42 @@ def sync_uuid_content(uuid)
...
       status, headers = result
       status == 200 && html?(headers) && !attachment?(headers)
     end
 
-    def collect_response(body)
+    def collect_content(chunks)
       content = nil
-      if body.respond_to?(:each)
-        body.each do |chunk|
+      if chunks.respond_to?(:each)
```
Comment (Tonkonozhenko) [inline lib/talkable/middleware.rb]: I think we don't need this check

Comment (vitalikdanchenko) [inline lib/talkable/middleware.rb]: yes, it was my debugging

File: lib/talkable/middleware.rb [hunk 3]
```diff
@@ -113,12 +113,8 @@ def modifiable?(result)
...
     def collect_content(chunks)
       content = nil
-      if chunks.respond_to?(:each)
-        chunks.each do |chunk|
-          content ? (content << chunk.to_s) : (content = chunk.to_s)
-        end
-      else
-        content = chunks
+      chunks.each do |chunk|
+        content ? (content << chunk.to_s) : (content = chunk.to_s)
```
Comment (Tonkonozhenko) [inline lib/talkable/middleware.rb]: Some simplification

``` ruby
content = ''
chunks.each { |chunk| content << chunk.to_s }
content
```

or 

``` ruby
''.tap do |content|
  chunks.each { |chunk| content << chunk.to_s }
end
```

File: lib/talkable/middleware.rb [hunk 4]
```diff
@@ -60,25 +77,44 @@ def sync_uuid_url(uuid)
...
+    end
+
+    def integration_script_content
+      %Q{
+<script>
+  window._talkableq = window._talkableq || [];
+  _talkableq.push(['init', {
+    site_id: '#{CGI.escape(Talkable.configuration.site_slug)}',
+    server: '#{CGI.escape_html(Talkable.configuration.server)}'
+  }]);
```
Comment (bogdan) [inline lib/talkable/middleware.rb]: `push(['init', #{init_parameters.to_json}])` this is the only safe way of doing that.

Comment (vitalikdanchenko) [inline lib/talkable/middleware.rb]: 👍

**Comment by Tonkonozhenko** [comment]
LG

**Pull request merged**